pkg/virt-controller/services/template_test.go

```diff
@@ -210,6 +211,30 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.Containers[0].VolumeMounts[1].MountPath).To(Equal("/var/run/kubevirt"))
 				Expect(*pod.Spec.TerminationGracePeriodSeconds).To(Equal(int64(60)))
 			})
+
+			It("should add node selector for node discovery feature to template", func() {
+
+				os.Setenv("FEATURE_GATES", "CPUNodeDiscovery")
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "testvmi",
+						Namespace: "default",
+						UID:       "1234",
+					},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Domain: v1.DomainSpec{
+							CPU: &v1.CPU{
+								Model: "Conore",
```

Comment:
typo: Conroe (probably irrilevant for the test as long as the string matches - but this kind stress the point I made above)